lib/trino-filesystem-cache-alluxio/src/main/java/io/trino/filesystem/alluxio/AlluxioFileSystemCache.java

```diff
@@ -94,4 +103,10 @@ protected URIStatus uriStatus(TrinoInputFile file, String key)
         String cacheIdentifier = hashFunction.hashString(key, UTF_8).toString();
         return new URIStatus(info, CacheContext.defaults().setCacheIdentifier(cacheIdentifier));
     }
+
+    private boolean uncacheable(TrinoInputFile file)
+    {
+        return denylist.stream()
+                .anyMatch(deny -> file.location().path().contains(deny));
```

Comment:
in the config description we say: `patterns` but here we have a substring matching.

`.anyMatch(pattern -> file.location().path().matches(pattern))`

Reply:
Excluding by file location string feels like the wrong level of abstraction to me. I don't think file locations have to necessarily contain schema or table name and it's inconvenient for users to hunt down file location for table.
Ideally this should work similar to `hive.file-status-cache-tables` where we identify targets for caching by either specifying `schema.*` or `schema.tablename`.